dds_registration/models.py

```diff
@@ -84,6 +84,10 @@ def sync_email_and_username(self):
             self._original_username = self.username
             # TODO: To do smth else if email has changed?
 
+    def get_full_name_or_email(self):
```

Comment:
I don't think this is right - we have to know the names of people for billing them, addressing them, etc. So I don't think we can allow people to register without providing their names.

Reply:
At the moment these values sometimes optional. Probably we have to make one more issue to fix that.

Reply:
Added #90.